joy2twist/include/joy2twist/joy2twist_node.hpp

```diff
@@ -74,6 +99,8 @@ class Joy2TwistNode : public rclcpp::Node
 static constexpr char FAST[]{"fast"};
 static constexpr char REGULAR[]{"regular"};
 static constexpr char SLOW[]{"slow"};
+
+static constexpr float AXIS_TO_BTN_DEADZONE = 0.05f;
```

Comment:
```suggestion
static constexpr char kFast[]{"fast"};
static constexpr char kRegular[]{"regular"};
static constexpr char kSlow[]{"slow"};

static constexpr float kAxisToButtonDeadzone = 0.05f;
```
As I mentioned before